src/Utilities.UnitTests/ToolTask_Tests.cs

```diff
@@ -679,22 +679,22 @@ public void ToolPathIsFoundWhenDirectoryExistsWithNameOfTool()
         [Fact]
         public void FindOnPathSucceeds()
         {
-            string expectedCmdPath;
+            string[] expectedCmdPath;
             string shellName;
             if (NativeMethodsShared.IsWindows)
             {
-                expectedCmdPath = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.System), "cmd.exe");
+                expectedCmdPath = new[] { Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.System).ToLower(), "cmd.exe") };
```

Comment:
nit:
You shouldn't have to do ToLower here, since supposed to be case insensitive—it doesn't seem like it's currently failing on Windows without it, but if we did want to fix it, I imagine changing the ShouldBe to take different casing would be a preferred fix.